cli/lfc/src/main/java/org/lflang/cli/Lfc.java

```diff
@@ -107,16 +108,22 @@ public class Lfc extends CliBase {
       description = "Specify the runtime scheduler (if supported).")
   private String scheduler;
 
-  @Option(
-      names = {"-t", "--threading"},
-      paramLabel = "<true/false>",
-      description = "Specify whether the runtime should use multi-threading" + " (true/false).")
-  private String threading;
+  /** Mutually exclusive options related to threading. */
+  static class ThreadingMutuallyExclusive {
+    @Option(
+        names = "--single-threaded",
+        arity = "0",
+        description = "Specify whether the runtime should be single-threaded.")
+    private boolean singleThreaded;
+
+    @Option(
+        names = {"-w", "--workers"},
+        description = "Specify the default number of worker threads.")
```

Comment:
How does this specify the "default" number of worker threads? What will be the actual number of worker threads? Shouldn't the "default" number of worker threads be the number you get if you do _not_ specify this option?

Reply:
True, I'll remove the word "default" the CLI description because it's misleading.

After the reading the documentation, it seems like we meant to say the `workers` property in the LF target statement defines the "default" number of threads, as opposed to the `--workers` CLI option which overrides the former if supplied. So either way, the CLI option shown here shouldn't be called "default".
Source: https://www.lf-lang.org/docs/handbook/target-language-details?target=c#multithreaded-implementation